eras/babbage/impl/src/Cardano/Ledger/Babbage/Rules/Utxow.hs

```diff
@@ -22,22 +22,17 @@ import Cardano.Ledger.Alonzo.Rules.Utxow
     witsVKeyNeeded,
   )
 import Cardano.Ledger.Alonzo.Scripts (Script)
-import Cardano.Ledger.Alonzo.Tx (ValidatedTx (..), wits)
+import Cardano.Ledger.Alonzo.Tx (ValidatedTx (..)) -- , wits)
```

Comment:
Forgotten comment
```suggestion
import Cardano.Ledger.Alonzo.Tx (ValidatedTx (..))
```

Reply:
good catch, that's embarrassing :grimacing: